tests/support/virtualenv.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
```

Comment:
We already have [this helper](https://github.com/saltstack/salt/blob/master/tests/support/helpers.py#L1620).